plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/ui/AppToolsTabGroup.java

```diff
@@ -0,0 +1,15 @@
+package com.google.cloud.tools.eclipse.appengine.localserver.ui;
+
+import org.eclipse.debug.ui.AbstractLaunchConfigurationTabGroup;
+import org.eclipse.debug.ui.ILaunchConfigurationDialog;
+import org.eclipse.debug.ui.ILaunchConfigurationTab;
+
+public class AppToolsTabGroup extends AbstractLaunchConfigurationTabGroup {
```

Comment:
I'm not sure using AppTools in the name is good. This is for local App Engine devserver launch config. It shouldn't refer to implementation details in my opinion (now that we got rid of the other runtimes).